MediaPlayer/MediaControls.js

```diff
@@ -202,7 +210,7 @@ const MediaControls = kind({
 
 	},
 
-	render: ({badge, menuIcon, nextTrackIcon, onNext, onPause, onPlay, onPrevious, onRepeat, onShuffle, paused, pauseIcon, playIcon, previousTrackIcon, repeatIcon, shuffle, shuffleIcon, ...rest}) => {
+	render: ({badge, disabled, menuIcon, nextTrackIcon, onNext, onPause, onPlay, onPrevious, onRepeat, onShuffle, paused, pauseIcon, playIcon, previousTrackIcon, repeatIcon, shuffle, shuffleIcon, ...rest}) => {
```

Comment:
What I don't understand is this is not the sandstone props. The structure is quite different from Sandstone.
Why are you just changing this?
Why Don't you add `spotlight-disabled`?
The title of the PR and the actual edit do not match. so I want know the reason

Reply:
I discussed this with the team. In short, we did not choose the proper title for this task/PR. It just focuses on adding the disabled and spotlightDisabled props for the MediaPlayer.
Sandstone does not have a stand-alone component for the MediaPlayer, only for VideoPlayer, so we just used the parts that corresponded to the Agate MediaPlayer design. 
If you guys consider that we should add extra functionality/props, just let us know.